components/atoms/divider/index.js

```diff
@@ -11,13 +11,13 @@ import { SIZE } from '../../'
 // Style
 import styled from 'styled-components'
 
-export const Divider = ({ content, size, style }) =>
-  <StyledDivider content={content} size={size} style={style}>
+export const Divider = ({ colour, content, size, style }) =>
+  <StyledDivider colour={colour} content={content} size={size} style={style}>
```

Comment:
@kheruc Please use the word context here, just whats its called in other components.